beacon-chain/blockchain/service.go

```diff
@@ -296,6 +295,16 @@ func (s *Service) StartFromSavedState(saved state.BeaconState) error {
 	if err := s.clockSetter.SetClock(startup.NewClock(s.genesisTime, vr)); err != nil {
 		return errors.Wrap(err, "failed to initialize blockchain service")
 	}
+
+	earliestAvailableSlot, custodySubnetCount, err := s.updateCustodyInfoInDB(saved.Slot())
```

Comment:
do we not need some kind of fulu enabled check for these?

Reply:
Fixed in f55631688ca33f4bc715c6569aec79810ebf73c2.